en/mkdocs.yml

```diff
@@ -63,6 +63,8 @@ nav:
       - Code navigation: edit-the-code/code-navigation.md
       - Code actions: edit-the-code/code-actions.md
       - Diagnostics: edit-the-code/diagnostics.md
+      - Commands: edit-the-code/commands.md
```

Comment:
Is this a new page, seems this PR doesn't have commands.md

Reply:
These are the content that was moved from the VS Code marketplace. The `commands.md` file is added via https://github.com/wso2/docs-ballerina-vscode-extension/pull/102.